hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -40,6 +40,8 @@
 class MatchWithDistance(t.TypedDict):
     content_id: int
     distance: str
+    signal_type: str
+    signal_value: str
```

Comment:
blocking: Let's leave this off in this pass, since this is trickier than it looks